be1-go/hub/socket_types.go

```diff
@@ -0,0 +1,56 @@
+package hub
+
+import (
+	"github.com/gorilla/websocket"
+	"sync"
+)
+
+// socket types
+type SocketType string
+
+const (
+	clientSocket SocketType = "client"
+	organizerSocket SocketType= "organizer"
+	witnessSocket SocketType= "witness"
+)
+
+func newSocket(socketType SocketType, h Hub, conn *websocket.Conn) *baseSocket {
```

Comment:
suggestion: `newBaseSocket` instead of `newSocket`

Reply:
Yes will do! Unfortunately I was not careful and was too quick to merge this PR but I will make sure to change this on my current branch.